### ISSUE TITLE
Feature Request: reload file list on cycle

### ISSUE THREAD

**Issue by PeteCozens** [derf/feh#213]
I’m using FEH in slideshow mode to power a photo frame on a Raspberry Pi and it’s doing the job magnificently. The only issue I’ve discovered during testing is that FEH appears to search for matching files only once at startup, meaning that it does not pick up any new/deleted files until such stage as it is restarted. In order to work around this I’ve had to setup a CRON job to periodically kill FEH and restart the job to pickup any updates in the list of files to display.

What would be really helpful (and an awful lot neater than my current solution) would be to have a parameter that tells FEH to reload the file list when it comes to the last image in a cycle, so that it has an updated list of files to use in the next one. 

Just a thought – thanks for your consideration.

**Comment by derf** [comment]
Try using --reload <something something>. feh will not only reload the current image, but also the entire filelist, so if you run e.g. "feh --reload 60 somedir/" it'll check for new files every 60 seconds.

However, note that filelist reloading is not well tested yet. If you run into bugs, please let me know.

**Comment by yournameistaken** [comment]
I have the same use case as Pete and the reload function doesn't work for a directory.  My workaround was to run feh with the reload option, but for a specific file.  I have a script that copies an image from the directory, and overwrites that specific file.  This does seem to work.  I do have an problem with this though, and will post it as another issue (displaying the image before it is fully loaded).

Also, the feh slideshow seems to hang if the directory is large (a few hundered images).
